src/App.tsx

```diff
@@ -15,71 +16,93 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+type State = {
+  selectedProduct: string;
+};
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+export class App extends React.Component<{}, State> {
+  state = {
+    selectedProduct: '',
+  };
 
-      {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+  addRemoveGood = (currentProduct: string) => {
+    if (this.state.selectedProduct === currentProduct) {
+      this.setState({ selectedProduct: '' });
+    } else {
+      this.setState({ selectedProduct: currentProduct });
+    }
+  };
```

Comment:
in next tasks try to avoid creating such common handlers
create separate and then conditional call it
it will help to maintain growing code